zeus/device/gpu.py

```diff
@@ -529,43 +547,70 @@ def setGpuLockedClocks(self, minGpuClockMHz: int, maxGpuClockMHz: int) -> None:
             self.handle,
             minGpuClockMHz,
             maxGpuClockMHz,
-            clk_type=amdsmi.AMDSMI_CLK_TYPE_GFX,
+            clk_type=amdsmi.AmdSmiClkType.GFX,
         )
 
     @_handle_amdsmi_errors
     def resetMemoryLockedClocks(self) -> None:
         """Resets the memory locked clocks of the specified GPU to their default values."""
-        amdsmi.amdsmi_reset_gpu_clk(
-            self.handle, clk_type=amdsmi.AMDSMI_CLK_TYPE_SYS
-        )  # TODO: check docs
+        # Get default MEM clock values
+        _, max_clk, min_clk = amdsmi.amdsmi_get_clock_info(
+            self.handle, amdsmi.AmdSmiClkType.MEM
+        )
+
+        amdsmi.amdsmi_set_gpu_clk_range(
+            self.handle,
+            min_clk,
+            max_clk,
+            clk_type=amdsmi.AmdSmiClkType.MEM,
+        )
 
     @_handle_amdsmi_errors
     def resetGpuLockedClocks(self) -> None:
         """Resets the GPU locked clocks of the specified GPU to their default values."""
-        amdsmi.amdsmi_reset_gpu_clk(
-            self.handle, clk_type=amdsmi.AMDSMI_CLK_TYPE_GFX
-        )  # TODO: check docs
+        # Get default GPU clock values
+        _, max_clk, min_clk = amdsmi.amdsmi_get_clock_info(
```

Comment:
Same question, aren't the return values a dictionary?
https://rocm.docs.amd.com/projects/amdsmi/en/latest/py-interface_readme_link.html#amdsmi-get-clock-info

Reply:
If it indeed returns a dictionary -- have you really tried calling each API on an actual GPU at least to make sure it doesn't raise a trivial error like this?

Reply:
I'm deferring merge before we make sure this is done.

Reply:
Yes, this is a mistake. This may have come up when I was resolving merge conflicts, not sure. I went through and fixed all issues, and retested to make sure every methods works